build.gradle

```diff
@@ -184,7 +184,7 @@ def versions = [
         restAssured                     : '3.0.7',
         piTest                          : '1.3.2',
         commonMark                      : '0.11.0',
-        probateCommons                  : '1.1.0',
+        probateCommons                  : 'DTSPB-535-Deathcert-flow3',
         jacksonDatabind                 : '2.9.8',
```

Comment:
Is it ok to merge to master with this as version number? What if another story affects the same module?